roombasim/ai/task.py

```diff
@@ -53,6 +78,11 @@ class Task(object):
     Subclasses must implement the update method and can optionally
     override the __init__ constructor.
     '''
+
+    # Task completion enum
```

Comment:
Did we want to have these as an enum or did we want to wrap them into their own `TaskState` class? Or are they only getting used internally?

Reply:
yeah this is a good idea, I'll create a TaskState class

Reply:
fixed